client/src/components/public/calendar/ConfirmEventType.vue

```diff
@@ -96,16 +97,8 @@ export default {
     components: {
         EventInfo
     },
-    data: () => ({
-        isReady: false,
-        formValid: false,
-        showPassword: false,
-        meetingFormData: {
-            name: '',
-            email: '',
-            additionalInfo: ''
-        },
-        emailRules: [
+    mounted() {
+        this.emailRules = [
```

Comment:
Using of vuelidate can save a lot of your time and make code of our project more consistent see SingUp.vue as example